integration-tests/integration/reviewer/view-report.spec.js

```diff
@@ -58,14 +55,14 @@ context('view review page', () => {
       })
     )
 
-    const allIncidentsPage = AllIncidentsPage.goTo()
-    allIncidentsPage.getTodoRows().should('have.length', 2)
-    allIncidentsPage.getNoCompleteRows().should('exist')
+    const notCompletedIncidentsPage = NotCompletedIncidentsPage.goTo()
+    notCompletedIncidentsPage.getTodoRows().should('have.length', 2)
+    notCompletedIncidentsPage.getNoCompleteRows().should('not.exist')
```

Comment:
this doesn't make sense as notCompletedIncidentsPage would never show "No Complete Rows".

Reply:
agreed